DOCUMENTATION.md

```diff
@@ -284,9 +291,73 @@ Revoke support access to an application.
 #### Examples:
     >> > resin.models.application.revoke_support_access('5685')
     'OK'
+## Build
+
+This class implements model for build for Resin Python SDK.
```

Comment:
nit: Could change this to `This class implements build model for Resin Python SDK.` to match the earlier `This class implements application model for Resin Python SDK.` phrasing?